src/features/contacts/components/contacts-management/contacts-management.tsx

```diff
@@ -29,6 +31,28 @@ export function ContactsManagement({
   const [searchName, setSearchName] = React.useState("")
   const debouncedSearchName = useDebounce(searchName)
   const { contacts, total } = useContactsList(debouncedSearchName)
+  const inputRef = useRef<HTMLInputElement | null>(null)
+
+  const { updateContact } = useContactsStore()
+  const importContacts = (changeEvent: ChangeEvent<HTMLInputElement>) => {
+    const reader = new FileReader()
+    reader.onload = onLoadEvent => {
+      const contacts: Contact[] = JSON.parse(
+        onLoadEvent.target?.result as string,
+      )
+      contacts.forEach(contact => updateContact(contact.address, contact))
```

Comment:
After the import is complete we should notify. Something like, "13 contacts imported" like we do on successful update.

Reply:
Done. Resolving.